### ISSUE TITLE
No issue, just a question

### ISSUE THREAD

**Issue by nogo10** [raineorshine/creatable#5]
In reading the docs, its clear what creatable is doing . However im somewhat new to frameworks, please advise if creatable can do what React does but in a simpler form for small SPA (single page applications)?

**Comment by nogo10** [comment]
If not, what about in combination with a tiny reactive frp like cellx?

**Comment by raineorshine** [comment]
Hi! Thanks for the questions. Creatable was created a long time ago, long
before React. React is I think the realization of the vision of Creatable.
It has a similar syntax when ReactDOM is used directly. React solves the
performance issue with immutability and virtual DOM diffing, a new
standard. And while React does not have anything to say about data binding
itself, frameworks like Redux and many others provide opinions about
managing data in cooperation with your view.

In short, I highly recommend React on any size project. In order to reflect
the terse, Javascript-centric syntax of Creatable, eschew JSX in favor of
ReactDOM (part of React) or Uber's r-dom (https://github.com/uber/r-dom).

**Comment by nogo10** [comment]
Forgive my ignorance, I was appreciating the simplicity of creatable ~~as a replacement for jade or pug in an Express environment. I was specifically looking on binding view with data.~~ I will read up on redux and ReactDOM. thnx

**Comment by raineorshine** [comment]
No problem! I'm glad you asked! Hopefully my explanation made sense. Good
luck!
